scripts/update_hook_revs.py

```diff
@@ -48,7 +49,7 @@ def get_repl_value(match, dependencies: dict[str, str], outdated_hooks: list[str
 
         # Use the v prefix if it was used before
         if ver.startswith("v"):
-            new_ver = ver[0] + new_ver
+            new_ver = f"v{new_ver}"
```

Comment:
Wouldn't this duplicate the `v` at the beginning?
I don't see anything in the `hook_rev` regex capture groups that would exclude it

Reply:
No, this cannot happen. The `new_ver` comes from the lock file, which uses semantic version numbers.

Reply:
Ok, then that makes more sense than before